### DISCUSSION TITLE
Error executing straingst kmersim --all-vs-all -t 4 -S jaccard -S subset strainge_db/*.hdf5 > similarities.tsv

### DISCUSSION THREAD

**Discussion by itika05** [broadinstitute/StrainGE#38] [Q&A]
Dear authors, 
I keep encountering an error while executing "straingst kmersim --all-vs-all -t 4 -S jaccard -S subset strainge_db/*.hdf5 > similarities.tsv". Following is the error message that I keep getting: "Traceback (most recent call last): File "/home/ubuntu/.local/bin/straingst", line 8, in <module> sys.exit(straingst_cli()) File "/home/ubuntu/.local/lib/python3.10/site-packages/strainge/cli/main.py", line 110, in call self.run(args) File "/home/ubuntu/.local/lib/python3.10/site-packages/strainge/cli/registry.py", line 83, in run rc = subcommand_func(**args_dict) File "/home/ubuntu/.local/lib/python3.10/site-packages/strainge/cli/kmersets.py", line 379, in call scores = list(pool.imap_unordered(runner, to_compute_iter, File "/usr/lib/python3.10/multiprocessing/pool.py", line 451, in <genexpr> return (item for chunk in result for item in chunk) File "/usr/lib/python3.10/multiprocessing/pool.py", line 873, in next raise value ZeroDivisionError: division by zero help me find the files and ignore which file is causing error" 
I was wondering if someone can please help me resolve the error. 
Thanks in advance!

**Comment by m-nikolaidis** [comment]
Hello, I'm not affiliated with the StrainGE development but encountered the ZeroDivision Error (slightly different overall error message). The problem was with the file Esch_coli_KE47 because kmerize command wasn't working on that.

Possibly the E. coli Zagreb strains may be causing problems because they were pointing to broken links in my machine and had to manually fix them.
